pyqtgraph/graphicsItems/ViewBox/ViewBox.py

```diff
@@ -1543,11 +1543,12 @@ def updateViewRange(self, forceX=False, forceY=False):
                     link.linkedViewChanged(self, ax)
 
             # emit range change signals
+            # print('announcing view range changes:',self.state['viewRange'] )
             if changed[0]:
                 self.sigXRangeChanged.emit(self, tuple(self.state['viewRange'][0]))
             if changed[1]:
                 self.sigYRangeChanged.emit(self, tuple(self.state['viewRange'][1]))
-            self.sigRangeChanged.emit(self, self.state['viewRange'])
+            self.sigRangeChanged.emit(self, self.state['viewRange'], changed)
```

Comment:
ViewBox has already tracked which axis was changed in this update, and now includes that in the signal.